### DISCUSSION TITLE
EDGE浏览器存在乱码功能

### DISCUSSION THREAD

**Discussion by Index-z** [the1812/Bilibili-Evolved#3223] [综合讨论 / General]
EDGE浏览器安装最新的2.1.7插件后存在不少乱码的情况。
![QQ截图20220406184401](https://user-images.githubusercontent.com/101051902/161957893-22fe61f7-b38d-4c3c-b0d6-b373ffe99d79.png)
![QQ截图20220406184415](https://user-images.githubusercontent.com/101051902/161957898-07c78aa8-0985-4045-a9c0-0c65c2f2354c.png)

**Comment by Geeyun-JY3** [comment]
看看这个：#3207

**Reply by Index-z** [reply]
还需要更换插件才能解决，有点麻烦 这个Tampermonkey里面装了不少脚本 凑合用吧  看后面版本会不会修复了
同样的版本在opera浏览器里面都正常，但是opera同步有点不好用 想着EDGE的同步方便点，结果发现在EDGE里面就有乱码了
